src/components/DeployedContractsTable/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import CONTRACT_VERSIONS from './contracts-versions.json';
+
+const {
+  LSP23LinkedContractsFactory,
+  UniversalProfileInitPostDeploymentModule,
+  UniversalProfileInit,
+  LSP6KeyManagerInit,
+  LSP1UniversalReceiverDelegateUP,
+} = CONTRACT_VERSIONS;
+
+const BaseContractsRows = () => {
+  const baseContracts = [
+    UniversalProfileInit,
+    LSP6KeyManagerInit,
+    LSP1UniversalReceiverDelegateUP,
+  ];
+
+  const baseContractNames = Object.keys({
+    UniversalProfileInit,
+    LSP6KeyManagerInit,
+    LSP1UniversalReceiverDelegateUP,
+  });
+
+  return baseContracts.map((baseContract, index) => {
+    return baseContract.versions.map(({ version, address, releaseurl }) => (
+      <>
+        <tr>
+          <td>
+            <code>{baseContractNames[index]}</code>
+          </td>
+          <td>
+            <code>
+              <a href={releaseurl} target="_blank">
+                {version}
+              </a>
+            </code>
+          </td>
+          <td>
+            <code>
+              <a
+                href={`https://explorer.execution.mainnet.lukso.network/address/${address}`}
+                target="_blank"
+              >
+                {address}
+              </a>
+            </code>
+          </td>
+        </tr>
+      </>
+    ));
+  });
+};
+
+export default function DeployedContractsTable() {
+  const { address: lsp23Address } = LSP23LinkedContractsFactory;
+  const { address: postDeploymentModuleAddress } =
+    UniversalProfileInitPostDeploymentModule;
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <td>Contract</td>
```

Comment:
You need to use th here instead

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th